Patches/Core/ThingDefs_Races/Races_Animal_Temperate.xml

```diff
@@ -620,11 +550,4 @@
 		</value>
 	</Operation>
 
-	<Operation Class="PatchOperationReplace">
-		<xpath>Defs/PawnKindDef[defName="Tortoise"]/combatPower</xpath>
-		<value>
-			<combatPower>20</combatPower>
```

Comment:
Vanilla combat power is 70, meaning one tortoise is equivalent to a little less than a lynx (80), which I think is kind of absurd. I'd probably leave this patch as it is.